pronto_mia/ios/Runner/Assets.xcassets/LaunchBackground.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "background.png",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
```

Comment:
Don't you have to include a different file on 2x and 3x like you did in `LaunchImage.imageset/Contents.json`?

Reply:
The background.png is a 1x1 white tile that gets repeated as often as necessary. I don't think an adjustment is necessary. this file was auto-generated by flutter.